runtime/doc/vim_diff.txt

```diff
@@ -21,9 +21,9 @@ these differences.
 ==============================================================================
 1. Configuration					    *nvim-configuration*
 
-- Use ".nvimrc" instead of ".vimrc" for storing configuration.
-- Use ".nvim" instead of ".vim" to store configuration files.
-- Use ".nviminfo" instead of ".viminfo" for persistent session information.
+- Use `.nvimrc` instead of `.vimrc` for storing configuration.
+- Use `.nvim` instead of `.vim` to store configuration files.
+- Use `.nviminfo` instead of `.viminfo` for persistent session information.
```

Comment:
vimdoc has supported backticks for a long time, though it's odd it is not mentioned in `:help help`. We should start using these for technical identifiers instead of quotation marks, in most cases.